### PR TITLE
maintainers: add cigrainger

### PR DESCRIPTION
**Pull request by cigrainger**
###### Description of changes

Adding myself to the maintainers list, as per https://github.com/NixOS/nixpkgs-committers/issues/60#issuecomment-1065525515. cc @samuela.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by samuela — APPROVED** [review]

**Comment by samuela** [comment]
LGTM I'll merge once all the checks pass. It may take some time for the bot to propagate and for you to get added as a team member, but once that's done I'll add you to @NixOS/cuda-maintainers. Don't hesitate to ping me if that doesn't happen after a day or two!

File: maintainers/maintainer-list.nix
Comment (cole-h) [inline maintainers/maintainer-list.nix:14115]: Sorry to be so nitpicky, but: despite there being some other entries ignoring this request, please keep the list alphabetically sorted: https://github.com/NixOS/nixpkgs/blob/56a682817d68050298b9fb0c56628b5a85441ad4/maintainers/maintainer-list.nix#L49

Comment (samuela) [reply maintainers/maintainer-list.nix:14115]: oh yeah sorry I missed that! we should prob add a lint for this at some point

Comment (cigrainger) [reply maintainers/maintainer-list.nix:14115]: I'll make the change. 👍

Comment (cigrainger) [reply maintainers/maintainer-list.nix]: All done, sorry about that!

**Review by cole-h — APPROVED** [review]
Thanks ❤️

**Review by samuela — APPROVED** [review]

**Comment by samuela** [comment]
Ok, all checks passed. Went ahead and merged.

**Comment by cole-h** [comment]
FWIW, it appears @cigrainger is now a member of the maintainers team (welcome!) and can now either "Request to Join" the team (at https://github.com/orgs/NixOS/teams/cuda-maintainers/members), or be invited to it :)

**Comment by samuela** [comment]
Roger that, @cigrainger now added to @NixOS/cuda-maintainers! Welcome to the team!

**Pull request merged**
